realm/realm-library/src/main/java/io/realm/RealmObject.java

```diff
@@ -354,6 +365,30 @@ public final boolean load() {
     }
 
     /**
+     * Adds a change listener to a RealmObject.
```

Comment:
what about ""Adds a change listener to a RealmObject that will be triggered if any value field or referenced RealmObject field is changed, or the `RealmList` field itself is changed.` -- since change of the object in the `RealmList` field will not trigger the notification.